src/providers/injectedProviders.ts

```diff
@@ -70,6 +70,10 @@ const injectedProviders = [
  * @returns updated array of providers
  */
 export const checkRLoginInjectedProviders = (providers: IProviderUserOptions[]) => {
+  // Remove liquality injected provider
+  providers.forEach((provider) => {
+    if (provider.name === 'Liquality') providers.splice(providers.indexOf(provider), 1)
+  })
```

Comment:
Why are you excluding it? If the user is still using Liquality, for whatever reason, that should be their decision not the developers. Why not just allow it to be shown in the bottom with the other misc providers?